packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -896,6 +913,25 @@ class StackViewport extends Viewport implements IStackViewport {
     triggerEvent(this.element, Events.CAMERA_MODIFIED, eventDetail);
   }
 
+  private setVOILUTFunction(
+    voiLUTFunction: VOILUTFunctionType,
+    suppressEvents?: boolean
+  ): void {
+    if (this.useCPURendering) {
+      throw new Error('VOI LUT function is not supported in CPU rendering');
+    }
+
+    // make sure the VOI LUT function is valid in the VOILUTFunctionType which is enum
+    if (Object.values(VOILUTFunctionType).indexOf(voiLUTFunction) === -1) {
```

Comment:
This logic is used 3 times `Object.values(VOILUTFunctionType).indexOf(voiLUTFunction)` maybe refactor?

Reply:
This check can probably be removed actually since the `VOILUTFunctionType` is forced in the function signature.

Reply:
The other places are due to CSWIL passing in a string which needs to be properly checked. Once we convert to typed CSWIL these logic can be removed.